x-pack/test/functional/services/transform/wizard.ts

```diff
@@ -76,6 +76,12 @@ export function TransformWizardProvider({ getService }: FtrProviderContext) {
       await testSubjects.existOrFail(selector);
     },
 
+    async assertPivotPreviewChartHistogramButtonExists() {
```

Comment:
I think this name can be a bit confusing since we're actually checking that the button does NOT exist. I'd suggest a rename to make it more clear and add a second method (or rename it back) as soon as the button is available.

Reply:
Fixed in 1bc0bca.